### PR TITLE
Tclreadline recipe

### PR DESCRIPTION
**Pull request by abretaud**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by abretaud** [comment]
It should be ready now, ping @conda-forge/staged-recipes

File: recipes/tclreadline/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/flightaware/tclreadline/archive/v{{ version }}.tar.gz
+  sha256: a64e0faed5957b8e1ac16f179948e21cdd6d3b8313590b7ab049a3192ab864fb
+
+build:
+  number: 0
+  skip: true  # [win or osx]
```
Comment (chrisburr) [inline recipes/tclreadline/meta.yaml]: Have you tried building for macOS? It normally just works
```suggestion
  skip: true  # [win]
```

Comment (abretaud) [reply recipes/tclreadline/meta.yaml]: Yep, I tried but it failed, and I have no idea why, here's the log: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=104261&view=logs&j=c572e5d5-7ba7-55ce-5575-7ce1d9f7161e&t=81967b5f-0283-5c0b-87c1-e8fdeef842a3

Comment (chrisburr) [reply recipes/tclreadline/meta.yaml]: Looks like it's finding the headers from conda-forge but linking to the library from the macOS SDK when running autotools. Let's see if this fixes it.

Comment (chrisburr) [reply recipes/tclreadline/meta.yaml]: Looks like it's finding the headers from conda-forge but linking to the library from the macOS SDK when running autotools. Let's see if this fixes it.

**Pull request merged**